components/script/dom/htmltableelement.rs

```diff
@@ -51,6 +54,31 @@ impl HTMLTableElement {
 }
 
 impl HTMLTableElementMethods for HTMLTableElement {
+    // https://html.spec.whatwg.org/multipage/#dom-table-rows
+    fn Rows(&self) -> Root<HTMLCollection> {
+        #[allow(moved_no_move)]
```

Comment:
My suspicion of why it's crashing is this here. `RootedVec` isn't supposed to be moved, but the `TableRowFilter` is moved to a new `HTMLCollection` below. I had to add this lint in order to silence the compiler error.